CIP-0106/README.md

```diff
@@ -0,0 +1,152 @@
+---
+CIP: 106
+Title: Web-Wallet Bridge - Mutlisig wallets
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors: NA
```

Comment:
Since what you list as a Reference Implementation appears to be a working wallet, you should remove `NA` and list yourself (if alone) or BroClanWallet or its organisation (if this is a group effort) as an Implementor.

Reply:
resolving; fixed by https://github.com/cardano-foundation/CIPs/pull/617/commits/ce033b7d29946956417015b9efe8f3af14d06576